pkg/deployment/agent.go

```diff
@@ -100,25 +106,25 @@ func (a *Agent) Get() *appsv1.DaemonSet {
 						Args:  args,
 						Ports: []corev1.ContainerPort{
 							{
-								ContainerPort: 5775,
-								HostPort:      5775,
+								ContainerPort: zkCompactTrft,
+								HostPort:      zkCompactTrft,
```

Comment:
Nothing against this name, but for the record, I used "zk-compact-trft" as the name because of size limitations in the name field for the port.